internal/engine/netx/archival/archival.go

```diff
@@ -431,16 +431,16 @@ func NewDNSQueriesList(begin time.Time, events []trace.Event, dbpath string) []D
 func (qtype dnsQueryType) ipoftype(addr string) bool {
 	switch qtype {
 	case "A":
-		return strings.Contains(addr, ":") == false
+		return !strings.Contains(addr, ":")
 	case "AAAA":
-		return strings.Contains(addr, ":") == true
+		return strings.Contains(addr, ":")
```

Comment:
These were `staticcheck` warnings that I fixed. While reading again this diff, I've found that the tests for this function could be improved, so I rewrote them in a better way (see below).